### PR TITLE
Feature: Dedicated Host Local Disks

### PR DESCRIPTION
**Pull request by deepaksibm**
Screen Shot 2021-04-01 at 3 15 58 PM
Screen Shot 2021-04-01 at 3 37 23 PM
Screen Shot 2021-04-01 at 3 39 12 PM
Screen Shot 2021-04-01 at 3 54 36 PM
Screen Shot 2021-04-01 at 3 58 17 PM
Screen Shot 2021-04-01 at 3 59 25 PM
Screen Shot 2021-04-01 at 4 01 23 PM
Screen Shot 2021-04-01 at 4 01 56 PM
Screen Shot 2021-04-01 at 4 03 19 PM
Screen Shot 2021-04-01 at 4 04 24 PM
Screen Shot 2021-04-01 at 4 09 08 PM
Screen Shot 2021-04-01 at 4 16 56 PM

### CODE REVIEW

**Comment by deepaksibm** [comment]
Test cases are executed in production environment

**Comment by deepaksibm** [comment]
Terraform Show 
https://ibm.box.com/s/j02gs0no3d3hli9461mbi7reqdvcbm8h

File: ibm/data_source_ibm_is_dedicated_host_disk.go [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+
+	"github.com/IBM/vpc-go-sdk/vpcv1"
+)
+
+func dataSourceIbmIsDedicatedHostDisk() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceIbmIsDedicatedHostDiskRead,
+
+		Schema: map[string]*schema.Schema{
+			"dedicatedhost": &schema.Schema{
```
Comment (hkantare) [inline ibm/data_source_ibm_is_dedicated_host_disk.go:22]: change to dedicated_host we followed same reference in other related resources/datasource

File: ibm/data_source_ibm_is_dedicated_host_disks.go [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+
+	"github.com/IBM/vpc-go-sdk/vpcv1"
+)
+
+func dataSourceIbmIsDedicatedHostDisks() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceIbmIsDedicatedHostDisksRead,
+
+		Schema: map[string]*schema.Schema{
+			"dedicatedhost": &schema.Schema{
```
Comment (hkantare) [inline ibm/data_source_ibm_is_dedicated_host_disks.go:23]: same as above

**Pull request merged**